.travis.yml

```diff
@@ -49,6 +49,7 @@ matrix:
   fast_finish: true
 env:
   global:
+  - GRADLE_VERSION=4.10
```

Comment:
Is this really needed? It's going to cause maintenance issues when we upgrade gradle as it's yet another area one must remember to update.